src/Twig/Handler/RecordHandler.php

```diff
@@ -313,4 +314,26 @@ public function selectField($content, $fieldName, $startempty = false, $keyName
 
         return $retval;
     }
+
+    /**
+     * Return whether or not a given path resolves, i.e. if it has a defined route.
+     *
+     * @param string $name
+     * @param array  $parameters
+     * @param bool   $relative
+     *
+     * @return bool
+     */
+    public function isPath($name, $parameters = array(), $relative = false)
+    {
+        $success = true;
+
+        try {
+            $path = Library::path($name, $parameters, $relative);
```

Comment:
This is introducing a new dependency on a deprecated class/function.

Would be much better to use the SP directly.